RELEASES.md

```diff
@@ -4,10 +4,10 @@ Version 1.33.0 (2019-02-28)
 Language
 --------
 - [You can now use the `cfg(target_vendor)` attribute.][57465] E.g.
-  `#[cfg(target_vendor="linux")] fn main() { println!("Hello Linux!"); }`
+  `#[cfg(target_vendor="apple")] fn main() { println!("Hello Apple!"); }`
```

Comment:
Looks like more spurious changes? Shouldn't this (have been) fixed before approving...?

Reply:
This was intentional (note the separate commit); we wanted to land some additional release notes and since we were merging this into stable combined it into one.

In hindsight, I should've mentioned this in the PR description.